backend/src/graphql/resolvers-types.ts

```diff
@@ -28,22 +29,41 @@ export type Response = {
   message?: Maybe<Scalars['String']>;
 };
 
-export type Report = {
-  __typename?: 'Report';
+export type FeedbackResponse = {
+  __typename?: 'FeedbackResponse';
+  id: Scalars['Int'];
+  rating: Scalars['Int'];
+  items?: Maybe<Array<Maybe<Scalars['String']>>>;
+  comment?: Maybe<Scalars['String']>;
+  feedbackBase?: Maybe<FeedbackBase>;
+  feedbackBaseId?: Maybe<Scalars['Int']>;
+  createdAt: Scalars['DateTime'];
+  updatedAt: Scalars['DateTime'];
+};
+
+export type FeedbackBase = {
+  __typename?: 'FeedbackBase';
+  id: Scalars['Int'];
   title?: Maybe<Scalars['String']>;
-  createdAt?: Maybe<Scalars['String']>;
+  feedbackUid: Scalars['String'];
+  reportUid: Scalars['String'];
+  feedbackLink: Scalars['String'];
+  reportLink: Scalars['String'];
+  createdAt: Scalars['DateTime'];
+  updatedAt: Scalars['DateTime'];
+  feedbackResponses: Array<FeedbackResponse>;
```

Comment:
hmmmm just as a side note. I see a cycle here. Feedbackbase and FeedbackResponse include each other. Is this circular dependence ok? Probably it is :)

Reply:
Yeah, actually it is possible to get the Feedbackbase from the FeedbackResponse and vice versa in the Graphql queries. We can limit the levels though. I think we might actually have this in the backlog already.